docs/sources/pip.md

```diff
@@ -0,0 +1,11 @@
+# Pip
+
+The pip source uses `pip` CLI commands to enumerate dependencies and properties. It is expected that `pip` is available in the PATH before running `licensed`
```

Comment:
Is `pip` being available on the PATH needed?  It looks like the source uses `pip` from the virtual env.

Reply:
Can you list the `requirements.txt` file as a requirement for the source being enabled?

Reply:
>Is pip being available on the PATH needed? It looks like the source uses pip from the virtual env.

Reply:
> I was thinking about a case where this could be run without a virtual env set

🤔 I guess in that case, you could instead have the configuration setting point to the pip binary location than the virtual env folder.  Then, if the configuration setting is blank licensed would look for `pip` rather than `configured/path/to/pip`

Reply:
@jonabc Thinking about this a little more last night, I think I'd like to leave it as is and have the `venv` directory be the dependency as opposed to the pip binary. The reasons being
* `venv` workflow is pretty standard
* Having everything setup via the virtualenv eliminates the need for the script to handle `pip` args etc
* I don't think `pip` needs to be in the `PATH` since having the `venv` dir specified will derive where the pip binary is located and that should be sufficient.